src/Illuminate/Foundation/Testing/TestResponse.php

```diff
@@ -901,6 +901,10 @@ public function assertSessionDoesntHaveErrors($keys = [], $format = null, $error
             return $this->assertSessionMissing('errors');
         }
 
+        if (is_null($this->session()->get('errors'))) {
+            return PHPUnit::assertTrue(true);
```

Comment:
`assertTrue` doesn't returns anything. If you want to conform to the signature you'll have to adjust this to

```php
	        if (is_null($this->session()->get('errors'))) {
	            PHPUnit::assertTrue(true);

	            return $this;
            }
```